src/frontend/js/components/ReferralDetail/index.tsx

```diff
@@ -309,6 +331,35 @@ export const ReferralDetail: React.FC = () => {
                     </>
                   ) : null}
                 </span>
+                <span>
+                  {canCloseReferral ? (
+                    <>
+                      <button>
+                        <svg
```

Comment:
I think it would be clearer to add a separator (`<span>•</span>`) after the pen, and plainly spell out "Close" ("Clôturer" en français?) as the link, without an icon.